content/lang/fr/index.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: home
+title: Preact
+show_title: false
+toc: false
+---
+
+
+<jumbotron>
+    <h1>
+        <logo height="1.5em" title="Preact" text>Preact</logo>
+    </h1>
+
+    <p>Alternative à React rapide pesant 3ko et proposant la même API ES6.</p>
+
+    <p>
+        <a href="/guide/getting-started" class="home-button">Pour commencer</a>
+        <span class="home-button-sep">&nbsp; • &nbsp;</span>
+        <a href="/guide/switching-to-preact" class="home-button">Passer à Preact</a>
+    </p>
+    <p>
+        <github-stars user="developit" repo="preact">5,000+</github-stars>
+    </p>
+</jumbotron>
+
+
+<section class="home-top">
+    <h1>Une blibliothèque différente.</h1>
+</section>
+
+
+<section class="home-section">
+    <img src="/assets/home/metal.svg" alt="metal">
+
+    <h2>Plus proche du DOM</h2>
+    
+    <p>
+        Preact est une couche abstraîte du DOM conçue pour être la plus légère possible.
+        Le web est une plateforme stable, il est temps d'arrêter de le réimplémenter au nom de la sécurité.
+    </p>
+
+    <p>
+        Preact fait partie intégrante de la la plateforme web. Elle fait le différenciel entre le DOM Virtuel et le DOM réel, enregistre de vrais gestionnaires d'évènements, et s'intègre aussi parfaitement avec les autres bibliothèques.
+    </p>
+</section>
+
+
+<section class="home-section">
+    <img src="/assets/home/size.svg" alt="size">
+
+    <h2>Taille Réduite</h2>
+    
+    <p>
+        La plupart des frameworks UI sont tellement gros qu'ils comptent pour presque la totalité de la taille d'una application JavaScript.
+        Preact est différent: sa taille se fait oubliée au profit de <em>votre code</em> qui représente la majeure partie de votre application.
+    </p>
+    
+    <p>
+        Cela signifie moins de JavaScript à télécharger, parser et exécuter - laissant plus de temps à votre code, vous permettant de concevoir l'expérience que vous souhaitez sans vous battre avec un framework dont vous n'avez pas le contrôle.
+    </p>
+</section>
+
+
+<section class="home-section">
+    <img src="/assets/home/performance.svg" alt="performance">
+
+    <h2>Performance Accrue</h2>
+    
+    <p>
+        Preact est rapide, et pas seulement à cause de sa taille. C'est l'une des bibliothèques de DOM Virtuel les plus rapides du marché, grâce à une implémentation du différenciel simple et prédictible.
+    </p>
+    
+    <p>
+        Elle inclut même quelques améliorations de performances en plus comme la mise à jour en lots personnalisable, un rendu asynchrône optionnel, la réutilisation du DOM, et une gestion des évènements optimisée via [Linked State](/guide/linked-state).
+    </p>
+</section>
+
+
+<section class="home-section">
+    <img src="/assets/home/portable.svg" alt="portable">
+
+    <h2>Portable &amp; Embarquable</h2>
+    
+    <p>
+        La petite taille de Preact permet d'embarquer le paradigme du Composant DOM Virtuel dans des environnements où c'était jusque là impossible.
```

Comment:
I’d say: `La taille réduite de Preact permet d’embarquer le paradigme`**`de Composant du DOM Virtuel`**`dans des environnements `*`où l’on ne pouvait pas le faire jusque-là.`*
The end of the sentence is a bit messy but `où c'était jusque là impossible` doesn’t sound grammatically correct to me.